attractions/utils/datetime-utils.js

```diff
@@ -215,12 +215,12 @@ export function datesLessEqual(date1, date2) {
  */
 export function getCalendar(month, year, firstWeekday) {
   const calendar = [];
-  const dayCursor = new Date(0);
-  dayCursor.setFullYear(year, month);
+  const dayCursor = new Date();
+  dayCursor.setFullYear(year, month, 1);
 
   // Offset the start of the month to the closest left `firstWeekday`
   dayCursor.setDate(
-    1 - ((daysInWeek + dayCursor.getDay() - firstWeekday) % daysInWeek)
+    0 - ((dayCursor.getDay() + firstWeekday) % daysInWeek)
```

Comment:
I understand the motive behind removing `daysInWeek`, it does seem redundant with the modulo, but it is actually needed due to the way JavaScript calculates the modulo of negative numbers (which remains negative). Also, why did you change the `1` into `0` and `-` into `+`? It now marks today (15th of February) as Thursday instead of Monday for me (I currently live in UTC+3 timezone, btw).

![image](https://user-images.githubusercontent.com/11016151/107913916-5f414500-6f72-11eb-8a23-3dd327d78ccf.png)

Reply:
This change was erroneous -- a blind attempt to match the dates to the weekday labels that displayed on my screen. 

Your comment pointing out that my week started on Friday clued me in that the labels on my screen were also being calculated wrongly. You also pointed out that getWeekdays also used new Date(0). Once I updated getWeekdays to use new Date(1970, 0, 1), the labels matched up and I saw that the change to the logic here wasn't warranted.

Reply:
Cool! Thanks for **your** help in finding and fixing bugs!